bot.py

```diff
@@ -95,6 +98,42 @@ def chastise(self, inc: bool, channel_id: str):
         else:
             self._send_encouragement_message(channel_id)
 
+    def display_leaderboards(self, args: str, channel_id: str):
+        """
```

Comment:
You should be able to put this docstring on one line.

Reply:
Addressed in 573ab863ac319d8900e15645b26f0f84c1145133!